CHANGELOG

```diff
@@ -1,6 +1,8 @@
 devel
 -----
 
+* Fixed issue #17291: Server crash on error in the PRUNE expression.
```

Comment:
The backports have changelog entries with more details. :)

Reply:
Ok, will add the details here too